fastlane/Fastfile

```diff
@@ -381,4 +386,10 @@ SUPPORTED_LOCALES = [
     }
   end
 
+  private_lane :send_strings_for_translation do | options |
+    sh("cd .. && mkdir -p #{update_strings_path} && cp #{main_strings_path} #{update_strings_path} && git add #{update_strings_path}strings.xml")
+    sh("git diff-index --quiet HEAD || git commit -m \"Send strings to translation.\"")
```

Comment:
Double nitpick.

1. The verb "send" makes me thing this commit actually sends the strings, but if I understand the process correctly, that's a side effect. 
2. Git commit titles should not have a dot at the end. From the [Git SCM docs](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project/#_commit_guidelines): "_Write your commit message in the imperative: "Fix bug" and not "Fixed bug" or "Fixes bug."_'

```suggestion
    sh("git diff-index --quiet HEAD || git commit -m \"Update strings file for translation automation\"")
```